lib/src/kbasesearchengine/main/SearchInterface.java

```diff
@@ -0,0 +1,27 @@
+package kbasesearchengine.main;
+
+import kbasesearchengine.*;
+import kbasesearchengine.search.IndexingStorage;
+import workspace.WorkspaceClient;
+
+import java.util.Map;
+
+/**
+ * Created by umaganapathy on 1/26/18.
+ */
+public interface SearchInterface {
+
+    WorkspaceClient getWorkspaceClient();
+
+    IndexingStorage getIndexingStorage(String objectType);
```

Comment:
Is there a need for this method? Don't see it being used, but maybe I'm missing it

Reply:
It was there before. So i did not remove it.

Reply:
Go ahead and remove it if it's unused.

Reply:
It is unused. Removed it.